extensions/spark/install_spark.sh

```diff
@@ -128,7 +128,6 @@ export SPARK_DAEMON_MEMORY=${SPARK_DAEMON_MEMORY}m
 export SPARK_WORKER_DIR=${SPARK_WORKDIR}
 export SPARK_LOCAL_DIRS=${SPARK_TMPDIR}
 export SPARK_LOG_DIR=${SPARK_LOG_DIR}
-export SPARK_CLASSPATH=\$SPARK_CLASSPATH:${LOCAL_GCS_JAR}
```

Comment:
Okay sounds good. I just tested Spark standalone and streaming and they worked with SPARK_CLASSPATH. I'll push the change.